### PR TITLE
Feat/withdrawals reciept view

### LINKED ISSUE #135: Transaction receipt 2nd pass: Withdrawals
## Story

**As a** user who placed an withdrawal request 
**I want** to see all the key details of my transfer transaction in my browser wallet
**So that** I can be certain I am approving the correct transaction

## Acceptance Criteria

- [ ] The view contains the necessary data enrichment to provide
- [ ] Amount (with correctly formatted position decimals)
- [ ] From Key
- [ ] To Ethereum key

## Tasks

- [x] Detailed sketch
- [x] Visual design
- [ ] Frontend tasks:
    - [ ] Implement UI
    - [ ] Integrate with backend
    - [ ] Write unit tests
- [ ] Backend tasks:
    - [ ] TBC
    - [ ] Write unit tests
- [ ] Write end to end tests
- [ ] Code review
- [ ] Team verify acceptance criteria met
- [ ] Exploratory QA testing 
- [ ] Capture bugs and new edge cases in tickets (this one or new TBD) 
- [ ] Write automated regression tests for the A/Cs (expanding on A/Cs where possible) 
- [ ] Add story and A/Cs to spec doc 
- [ ] Review with stakeholders
- [ ] Release

## Sketch

image

image

### PR DESCRIPTION
**Pull request by dexturr**
Closes #135 

https://github.com/vegaprotocol/vegawallet-browser/assets/26136207/4eeab7ae-6403-4e51-82f8-d3ba18cf70b9

AC:
The view contains the necessary data enrichment to provide
 Amount (with correctly formatted position decimals)
 From Key
 To Ethereum key

### CODE REVIEW

**Comment by dalebennett1992** [comment]
image

Issue here that the asset name is being chopped by the scrollbar. Can we fix?

File: frontend/components/receipts/utils/string-amounts/amount-with-symbol.tsx [hunk 1]
```diff
@@ -4,10 +4,10 @@ export const locators = {
   priceWithSymbol: 'price-with-symbol'
 }
 
-export const PriceWithSymbol = ({ price, symbol }: { price: string | number; symbol?: string }) => {
+export const AmountWithSymbol = ({ amount, symbol }: { amount: string | number; symbol?: string }) => {
   return (
     <div data-testid={locators.priceWithSymbol}>
```
Comment (dexturr) [inline frontend/components/receipts/utils/string-amounts/amount-with-symbol.tsx:9]: classNames="flex items-centre flex-wrap"

**Comment by dexturr** [comment]
Screenshot 2023-10-05 at 20 14 08
